README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Tic Tac Toe Game
+
+> Tic-tac-toe Game is a console game for two players, X and O, who take turns marking the spaces in a 3×3 grid. The player who succeeds in placing three of their marks in a diagonal, horizontal, or vertical row is the winner. It is a solved game with a forced draw assuming best play from both players.
+
+![screenshot](assets/1.png)
+
+In order to win the game, a player must place three of their marks in a horizontal, vertical, or diagonal row.
+
+The following example game is won by the first player, X:
+
+Game of Tic-tac-toe, won by X
+Players soon discover that the best play from both parties leads to a draw. Hence, tic-tac-toe is most often played by young children, who often have not yet discovered the optimal strategy.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+In order to play or get a local copy you can clone this repository by using:
+
+git clone git@github.com:iliebabcenco/tic-tac-toe.git
+
+After cloning the repository, to play the game type ruby bin/main.rb in the terminal.
```

Comment:
[MANDATORY]

- [ ] Missing to tell the user to go to the repo directory or else ruby bin/main.rb won't work. You can add more specific instructions. Think of first-time GitHub users with no technical knowledge.

For example,
`
  cd tic-tac-toe
`